cypress/support/index.js

```diff
@@ -0,0 +1,21 @@
+// ***********************************************************
+// This example support/index.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+import './commands'
+
+beforeEach(() => {
+    cy.resetGame();
+    cy.visit('/');
+});
```

Comment:
After Belle and I have this files, do you think it'd be a good idea to put them to gitignore?

Reply:
I don't think the support folder should be ignored, as we may add more commands to `support/commands.js` and/or change the `beforeEach` or `afterEach` in this file

Reply:
Yeah okay, that's a good point